### PR TITLE
dataset info: present decoded IDs for admins and add peek

### PR DESCRIPTION
**Pull request by martenson**
ping @natefoo

### CODE REVIEW

**Comment by natefoo** [comment]
Record time from request to PR, thank you @martenson!

File: templates/show_params.mako [hunk 1]
```diff
@@ -174,11 +174,26 @@
         %if job:
             <tr><td>Tool Exit Code:</td><td>${ job.exit_code | h }</td></tr>
         %endif
-        <tr><td>History Content API ID:</td><td>${encoded_hda_id}</td></tr>
+        <tr><td>History Content API ID:</td>
```
Comment (bgruening) [inline templates/show_params.mako:177]: Should we also change the name to indicate that it is a decoded id for the admin?

Comment (martenson) [reply templates/show_params.mako:177]: I thought this is clear for people that are actually using IDs for anything. What would you suggest?

Comment (bgruening) [reply templates/show_params.mako:177]: Simply adding `(decoded)`? If this is obvious, all good.

Comment (martenson) [reply templates/show_params.mako:177]: I had it there, but @natefoo called it superfluous.

Comment (bgruening) [reply templates/show_params.mako:177]: ok :)

**Comment by mvdbeek** [comment]
I often use this as a starting point to interact with the API. Can we toggle this instead ?

**Comment by martenson** [comment]
@mvdbeek both are present

**Comment by mvdbeek** [comment]
D'oh. 👍 (I did look at the code this time).

**Comment by jmchilton** [comment]
Do we have a plan for replicating this via the API longer term? An admin only api endpoint to decode IDS maybe?

**Comment by martenson** [comment]
@jmchilton I would enhance the dict for admin using the same endpoint, is that a nono now?

**Comment by jmchilton** [comment]
@martenson That would be fine with me I think - I just wanted to know what the plan is.

**Comment by martenson** [comment]
@jmchilton I will implement that in a different PR

**Pull request merged**